dev_settings.py

```diff
@@ -0,0 +1,41 @@
+DEBUG = True
```

Comment:
Skoro mamy ten plik, to z `manage.py` można usunąć te `DEFAULT_SETTINGS` i dać `os.environ.setdefault("DJANGO_SETTINGS_MODULE", "dev_settings.py")`